resolwe_bio/tests/processes/test_reads_filtering.py

```diff
@@ -157,6 +157,91 @@ def test_hsqutils_trimm(self):
         self.assertFiles(hsqutils_trimm, 'fastq2', ['hsqutils_reads_trimmed_mate2.fastq.gz'],
                          compression='gzip')
 
+    def test_cutadapt_single(self):
+        inputs = {'src': ['cutadapt_single.fastq.gz', 'cutadapt_single1.fastq.gz']}
+        reads = self.run_process('upload-fastq-single', inputs)
```

Comment:
Use `prepare_reads(['cutadapt_single.fastq.gz', 'cutadapt_single1.fastq.gz'])`, similarly in other places.